packages/familie-header/src/header/Header.tsx

```diff
@@ -63,7 +63,7 @@ export const Bruker = ({ navn, enhet, popoverItems }: BrukerProps) => {
     );
 };
 
-const Header = ({ tittel, children, brukerinfo, tittelHref = '/', brukerPopoverItems }: HeaderProps) => {
+export const Header = ({ tittel, children, brukerinfo, tittelHref = '/', brukerPopoverItems }: HeaderProps) => {
```

Comment:
I think we should move the search input component to its own package. This way we can easily add or remove the search from our clients.

Reply:
The search component is customizable, i.e. you can add or remove it as you wish:
<header>
  <søk/>
</header>

Reply:
Awesome!